ports/dudelingsarcadesportsball/port.json

```
@@ -0,0 +1,29 @@
+{
+    "version": 1,
```

This should be version 3.
```suggestion
    "version": 3,
```